packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -142,6 +182,7 @@ const SelectedFilters = {
 const mapStateToProps = state => ({
 	components: state.components,
 	selectedValues: state.selectedValues,
+	componentProps: state.props,
```

Comment:
@mohdashraf010897 We don't want to re-render the component when `componentProps` would change. You can access the Redux store by using `$$store` property. Call `getState` on each demand, so you would get the fresh state.
![alt img](https://i.imgur.com/COVWqw6.png)

Reply:
👍